src/polymer/polymer2-config.ts

```diff
@@ -90,3 +86,29 @@ export function getMethods(node: estree.Node, document: JavaScriptDocument):
   }
   return methods;
 }
+
+export function getStaticMethods(
```

Comment:
Why is this in polymer2-config.ts if it's just about class static methods? Also document it?